frontends/web/src/routes/settings/settings.tsx

```diff
@@ -195,6 +207,21 @@ class Settings extends Component<Props, State> {
                           <FiatSelection />
                         </div>
                         <div className="column column-2-3">
+                          <div>
+                            <h3 className="subTitle">{t('settings.info.title')}</h3>
+                            <div className="box slim divide m-bottom-large">
+                              { version !== undefined && update !== undefined ? (
+                                <SettingsButton
+                                  optionalText={version}
+                                  secondaryText={update ? t('settings.info.out-of-date') : t('settings.info.up-to-date')}
+                                  icon = {update ? RedDot({}) : Checked({})}
+                                  hideButtonIcon={true}
+                                  onClick={ update ? () => apiPost('open', updatePath) : undefined }>
```

Comment:
I find this a bit more intuitive `onClick={() => update && apiPost('open', updatePath)}`

Reply:
Done :+1: